model/task_lifecycle.go

```diff
@@ -272,7 +272,7 @@ func TryResetTask(taskId, user, origin string, detail *apimodels.TaskEndDetail)
 			if t.DisplayOnly {
 				execTasks := map[string]string{}
 				for _, et := range t.ExecutionTasks {
-					execTask, err = task.FindOne(task.ById(et))
+					execTask, err = task.FindOneId(et)
```

Comment:
in this case we only get the execTask long enough to log statuses, so I don't think we need to merge test results